ui/apps/platform/src/types/policy.proto.ts

```diff
@@ -12,11 +12,13 @@ export type ListPolicy = {
 };
 
 // TODO supersedes src/Containers/Violations/PatternFly/types/violationTypes.ts
-export type PolicySeverity =
-    | 'LOW_SEVERITY'
-    | 'MEDIUM_SEVERITY'
-    | 'HIGH_SEVERITY'
-    | 'CRITICAL_SEVERITY';
+export const policySeverities = [
+    'LOW_SEVERITY',
+    'MEDIUM_SEVERITY',
+    'HIGH_SEVERITY',
+    'CRITICAL_SEVERITY',
+] as const;
```

Comment:
Oh, interesting, I hadn't used const assertion before so this was informative. The line below says `typeof policySeverities[number]` though. Will `PolicySeverity` not be a type of `string`? What is the benefit of using const assertion in this situation?

Reply:
Nope, `PolicySeverity` will be the exact same type it was before this change: a string literal union. The `as const` marks the array read only and can be used at runtime, while the `policySeverties[number]` can resolve to a union of the values in the array since it knows at compile time exactly what values the array can contain. Without the `as const` bit it would resolve `PolicySeverity` to be just a type of `string`.

Initially I thought this was kind of odd, since `policySeverities`  doesn't return a value at every index of `number`, but I think it works similarly to:
```typescript
type SomeType = Record<string, boolean>;
type OtherType = SomeType[keyof SomeType]; // type OtherType = boolean
```